Update browserbench.org for Speedometer 3.1 launch

rniwa opened the pull request:
#### 087118884f0b98dfd6f129e230315e89d58e9087
<pre>
Update browserbench.org for Speedometer 3.1 launch
<a href="https://bugs.webkit.org/show_bug.cgi?id=290743">https://bugs.webkit.org/show_bug.cgi?id=290743</a>

Reviewed by Anne van Kesteren.

Announce the release of Speedometer 3.1.

* Websites/browserbench.org/Speedometer3.1/package.json:
* Websites/browserbench.org/Speedometer3.1/resources/main.mjs:
* Websites/browserbench.org/announcements/index.html:
* Websites/browserbench.org/announcements/speedometer3.1/index.html: Added.

Canonical link: <a href="https://commits.webkit.org/292924@main">https://commits.webkit.org/292924@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/5a90d22396104d45b75b368dc4f7b4b4ba03f992

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/97491 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/17116 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/7332 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/102579 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/48020 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/17409 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/25568 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/74288 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/31468 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/100494 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/132/builds/13183 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/88166 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/54633 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/12952 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/47462 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/82982 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/6129 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/104598 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/24570 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/17923 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/83334 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/24942 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/84292 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/82755 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 vision-wk2~~](https://ews-build.webkit.org/#/builders/126/builds/27297 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/4986 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/18161 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/15744 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/24532 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/24354 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/27668 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/25928 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

Andrew-Fryer commented on Websites/browserbench.org/Speedometer3.1/package.json:
```diff
@@ -15,8 +15,8 @@
-        "dev": "http-server ./ -p 7000 -c-1 --cors",
-        "server": "http-server ./ -p 7000 --cors",
+        "dev": "http-server ./ -p 8080 -c-1 --cors",
+        "server": "http-server ./ -p 8080 --cors",
```
May I ask why the port change?

rniwa replied:
See https://github.com/WebKit/Speedometer/pull/500

Andrew-Fryer commented on Websites/browserbench.org/announcements/speedometer3.1/index.html:
```diff
@@ -0,0 +1,63 @@
+<p>In today’s minor update to the Speedometer benchmark, we’re fixing a handful of bugs:</p>
+
+<ul>
+    <li>In workloads simulating rendering of news sites using Next.js and Nuxt, we fixed a bug where a slightly different amount of work was done in different browsers. We also fixed a bug where we were constantly re-generating IDs for each item in a list, by explicitly specifying IDs for each item.</li>
```
Tiny nit: I think it should be: "list by" instead of "list, by".

rniwa replied:
Fixed.

Andrew-Fryer commented on Websites/browserbench.org/announcements/speedometer3.1/index.html:
```diff
@@ -0,0 +1,63 @@
+<h1>Announcing Speedometer 3.1</h1>
+<div class="byline">
+    <span class="author">by contributors from Apple, Google, Microsoft, and Mozilla</span>
+    <span class="date">March 30, 2025</span>
```
maybe update date?

rniwa replied:
Fixed.

annevk commented on Websites/browserbench.org/announcements/speedometer3.1/index.html:
```diff
@@ -0,0 +1,63 @@
+
+<p>Today, we are announcing an update to the Speedometer benchmark. This is a minor version bump fixing the benchmark harness to improve measurement accuracy.</p>
+
+<p>To recap, Speedometer 3.0 <a href="https://browserbench.org/announcements/speedometer3/">released in March 2024</a> was a breakthrough in browser benchmarking. The three major browser engines: Blink, Gecko, and WebKit collaborated under an <a href="https://github.com/WebKit/Speedometer/blob/main/Governance.md">open governance model</a> to create a benchmark designed to reflect the real-world W ...
```
```suggestion
<p>To recap, Speedometer 3.0 <a href="https://browserbench.org/announcements/speedometer3/">released in March 2024</a> was a breakthrough in browser benchmarking. The three major browser engines: Blink, Gecko, and WebKit collaborated under an <a href="https://github.com/WebKit/Speedometer/blob/main/Governance.md">open governance model</a> to create a benchmark designed to reflect the real-world Web. It has a wide variety of workloads ranging from to-do apps to charting, code &amp; WYSIWYG editing, and news sites. Together these workloads exercise a broad and representative cross section of browser engines, providing new opportunities to optimize different parts of browser engines. See <a href="https://browserbench.org/announcements/speedometer3/">Speedometer 3.0 announcement</a>s to learn more about the Speedometer benchmark.</p>
```

rniwa committed:
Update browserbench.org for Speedometer 3.1 launch
https://bugs.webkit.org/show_bug.cgi?id=290743

Reviewed by Anne van Kesteren.

Announce the release of Speedometer 3.1.

* Websites/browserbench.org/Speedometer3.1/package.json:
* Websites/browserbench.org/Speedometer3.1/resources/main.mjs:
* Websites/browserbench.org/announcements/index.html:
* Websites/browserbench.org/announcements/speedometer3.1/index.html: Added.

Canonical link: https://commits.webkit.org/292924@main
